src/main/groovy/com/sequenceiq/cloudbreak/client/CloudbreakClient.groovy

```diff
@@ -130,6 +133,19 @@ class CloudbreakClient {
         return response?.data?.id
     }
 
+    def String postRecipe(String recipe, Boolean publicInAccount) throws Exception {
```

Comment:
Is this method intended to create a recipe? In that case it is missing lots of fields from the JSON. The only case it is valid if you post the JSON as String in the recipe variable. Is that the case?